### PR TITLE
fix: Fixed build issues

### PR DESCRIPTION
**Pull request by anoopmd**
I was trying to run the monaco example. I ran into a couple of issues. This PR addresses them. Have also attached screenshots of error that I had enocuntered.
monaco-error

acorn-error

### CODE REVIEW

File: examples/monaco-graphql-webpack/babel.config.js [hunk 1]
```diff
@@ -1,14 +1 @@
-module.exports = {
```
Comment (anoopmd) [inline examples/monaco-graphql-webpack/babel.config.js:1]: We are using the root babel config for the `graphiql` package. This removal should be safe.
Plus it fixes the lint error.

File: examples/monaco-graphql-webpack/package.json [hunk 2]
```diff
@@ -12,17 +12,19 @@
...
-    "monaco-graphql": "file:../../packages/graphiql",
+    "monaco-graphql": "file:../../packages/monaco-graphql",
     "prettier": "^2.0.2"
   },
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "7.8.3",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/preset-env": "7.9.5",
     "@types/prettier": "^2.0.0",
+    "acorn": "^8.0.3",
```
Comment (anoopmd) [inline examples/monaco-graphql-webpack/package.json:23]: I ran `yarn` both in the root as well as the monaco example folder. I got the error saying that `acorn` path is missing and that it needs to be installed.

**Comment by anoopmd** [comment]
@acao Please take a look when you find some time. Thanks :)

**Comment by acao** [comment]
@anoopmd thanks for addressing this!

i wonder if this could be solved by adding the webpack monaco middleware package to the example devDependencies

i think this happened when i removed examples from the workspace

are you running `yarn` inside the example first before running `yarn start`?

**Comment by acao** [comment]
@anoopmd I resolved this with #1696, thanks for the heads up!

**Pull request closed without merging**